packages/central-server/src/apiV2/getCountryAccessList.js

```diff
@@ -1,9 +1,9 @@
-/**
- * Tupaia MediTrak
- * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
  */
 
-import { respond, reduceToDictionary } from '@tupaia/utils';
+import { respond } from '@tupaia/utils';
 
 const mapRequestsToEntities = (requestedEntities, projectCodeById) => {
```

Comment:
This is no longer used anywhere. Is it something that might be useful in future or should I just clean it up?

Reply:
I'm guessing the new project based endpoint supersedes this, so I think it's fine to remove